src/web/src/components/Posts/PostAvatar.tsx

```diff
@@ -58,11 +58,19 @@ const PostAvatar = ({ name, img, blog = '' }: AvatarProps) => {
       )
       .join('');
     return (
-      <a href={blog} className={classes.link}>
-        <Avatar className={classes.avatar}>
-          <p className={classes.text}>{initials}</p>
-        </Avatar>
-      </a>
+      <div>
+        {url.length > 0 ? (
```

Comment:
I'm not sure about this, but shouldn't we also be checking if url is not null as well, basically if it exists as well as if the length is greater than 0

Reply:
So, I think from what I understand after reading the `PostAvatar.tsx` component is that, if no `url` is given in the props, then the `url` is assigned to a `""` empty string. Either way, I can add it quickly it's no problem